specification/resources/reserved_ipv6/reservedIPv6s_create.yml

```
@@ -0,0 +1,47 @@
+operationId: reservedIPv6s_create
+
+summary: Create a New Reserved IPv6
```

```suggestion
summary: [Beta] Create a New Reserved IPv6
```

My understanding is that these are part of a closed beta. If that's the case, please label them appropriately so that we don't cause confusion for the majority of customers that do not have access.

---

Yes